src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -211,6 +218,8 @@ function MoneyRequestView({report, betas, parentReport, policyCategories, should
                             shouldShowRightIcon={canEdit && !isSettled}
                             titleStyle={styles.flex1}
                             onPress={() => Navigation.navigate(ROUTES.EDIT_REQUEST.getRoute(report.reportID, CONST.EDIT_REQUEST_FIELD.DISTANCE))}
+                            brickRoadIndicator={hasErrors && hasRouteError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : ''}
+                            error={hasErrors && hasRouteError ? lodashValues(ErrorUtils.getLatestErrorField(transaction, 'route'))[0] : ''}
```

Comment:
I don't think we should show the error message here, I think we should just set the brick road indicator. That's our normal pattern.

Reply:
Oh ok I see. That's fair.

Reply:
Ultimately I think we should use the RBR specifically for this flow because the errors should be shown on the individual waypoints that are invalid. However, that requires backend changes and can be handled later. I have an issue assigned for it.